ios/RNGeth.swift

```diff
@@ -122,14 +122,51 @@ class RNGeth: RCTEventEmitter, GethNewHeadHandlerProtocol {
             if let useLightweightKDF = config?["useLightweightKDF"] as? Bool {
                 nodeconfig.useLightweightKDF = useLightweightKDF
             }
-            if let ipcPath = config?["ipcPath"] as? String {
+            if let noDiscovery = config?["noDiscovery"] as? Bool {
+                nodeconfig.noDiscovery = noDiscovery
+            }
+            if let bootnodeEnodes = config?["bootnodeEnodes"] as? [String] {
+                guard let enodes = GethEnodes(bootnodeEnodes.count) else {
+                    throw error ?? RuntimeError("Unable to create GethEnodes")
+                }
+                for i in 0..<bootnodeEnodes.count {
+                    try enodes.set(i, enode: GethEnode(bootnodeEnodes[i]))
+                }
+                nodeconfig.bootstrapNodes = enodes
+            }
+            // HTTP RPC configurations, which should only be used for development & debugging
+            if let httpHost = config?["httpHost"] as? String {
                 // Workaround gomobile objc binding bug for properties starting with a capital letter in the go source
                 // See https://github.com/golang/go/issues/32008
                 // Once that bug is fixed the assertion will fail and we can switch back to:
-                // nodeconfig.ipcPath = ipcPath
+                // nodeconfig.httpHost = httpHost
+                nodeconfig.setValue(httpHost, forKey: "HTTPHost")
+            }
+            if let httpPort = config?["httpPort"] as? Int {
+                // See comment for httpHost
+                nodeconfig.setValue(httpPort, forKey: "HTTPPort")
+            }
+            if let httpVirtualHosts = config?["httpVirtualHosts"] as? String {
+                // See comment for httpHost
+                nodeconfig.setValue(httpVirtualHosts, forKey: "HTTPVirtualHosts")
+            }
+            if let httpModules = config?["httpModules"] as? String {
+                // See comment for httpHost
+                nodeconfig.setValue(httpModules, forKey: "HTTPModules")
+            }
+
+            if let ipcPath = config?["ipcPath"] as? String {
+                // See comment for httpHost
                 nodeconfig.setValue(ipcPath, forKey: "IPCPath")
                 assert(nodeconfig.ipcPath == ipcPath)
             }
+            if let logFile = config?["logFile"] as? String {
```

Comment:
:+1: I actually wanted to bring this up, I noticed it was happening only on android.